content/modules/sort/versions/1.10.1.um

```diff
@@ -0,0 +1,55 @@
+@version 1.10.1
+  @api Sort
+    @object hx.sort
+      @property compare [Function]
+        @param a [String/Number]
+        @param b [String/Number]
+        @returns Number
+        @bugfix
+          @issue 372
+          @description
+            Resolved an issue where sorting with undefined would treat it
+            as a string.
+
+      @property localeCompare [Function]
+        @param locale [String]
+        @param? options [Object]
+          @property nullsLast [Boolean]
+            @added
+              @issue 372
+              @description
+                Added an option for sorting nulls to the end of an array when
+                using localeCompare.
+            @description
+              Whether to sort nulls to the end of the array.
+            @default: false
+        @returns Function
+          @param a [String]
+          @param b [String]
+          @returns Number
+        @bugfix
+          @issue 372
+          @description
+            Resolved an issue where sorting with undefined would treat it
+            as a string.
+
+      @property compareNullsLast [Function]
+        @added
+          @issue 372
+          @description
+            Added an function for sorting nulls to the end of an array.
+        @description
+          A sort function that sorts nulls to the end of an array.
+
+        @extra
+          @@codeblock js
+            const array = [0, null, '1', undefined, 'string', 'abc']
+            array.slice(0).sort(hx.sort.compare)
+            // Returns: [0, "1", "abc", null, "string", undefined]
+
+            array.slice(0).sort(hx.sort.compareNullsLast)
+            // Returns: [0, "1", "abc", "string", null, undefined]
+
+        @param a [String/Number]: The first item to compare
+        @param b [String/Number]: The second item to compare
+        @returns Number: 1 when b should be before a, -1 when a should be before b and 0 when the terms match.
```

Comment:
Thank you very much! :-)
Can the return description be a bit more detailed, though? Something along the lines of
`like hx.sort.compare, but treating defined < null < undefined, returning 1, 0 or -1 accordingly.`